message_definitions/v1.0/common.xml

```diff
@@ -380,9 +380,6 @@
     </enum>
     <!-- enum of fence types to enable or disable as a bitmask -->
     <enum name="FENCE_TYPE" bitmask="true">
```

Comment:
Great, I've merged a slightly modified version of your https://github.com/mavlink/mavlink/pull/2248 to fix this upstream - can you pull that down?

Reply:
Done